legal-api/src/legal_api/reports/report.py

```diff
@@ -100,10 +100,11 @@ def _substitute_template_parts(template_code):
             'bc-annual-report/legalObligations',
             'bc-address-change/addresses',
             'bc-director-change/directors',
-            'certificate-of-incorporation/logo',
-            'certificate-of-incorporation/registrarSignature',
-            'certificate-of-incorporation/seal',
-            'certificate-of-incorporation/style',
+            'certificate-of-name-change/style',
+            'common/certificateLogo',
+            'common/certificateRegistrarSignature',
+            'common/certificateSeal',
+            'common/certificateStyle',
```

Comment:
Is certificateStyle common, or specific to certificate-of-incorporation? (I noticed that certificate-of-name-change has its own style.)

Reply:
certificateStyle is common for all certificates. certificateOfNameChange has some extra styles.

Reply:
Gotcha. Sounds good.